server.js

```diff
@@ -134,8 +134,7 @@ app.post('/webhook', function(req, res) {
 });
 
 var server = app.listen(process.env.PORT || 5000, function () {
-  var host = server.address().address;
   var port = server.address().port;
 
-  console.log('Example app listening at http://%s:%s', host, port);
+  console.log('Example app listening at http://localhost:%s', port);
```

Comment:
for me this was always reporting the address as `::` (weird) so I figured I'd just hard code localhost